doc/conseil.sql

```diff
@@ -328,6 +363,13 @@ CREATE INDEX ix_operations_source ON public.operations USING btree (source);
 
 CREATE INDEX ix_accounts_checkpoint_block_level ON public.accounts_checkpoint USING btree (block_level);
 
+--
+-- Name: ix_proposals_protocol; Type: INDEX; Schema: public; Owner: -
+--
+
+CREATE INDEX ix_proposals_protocol ON public.proposals USING btree (protocol_hash);
```

Comment:
What inspired this index?

Reply:
The most common query pattern on this table will be to get proposal details by protocol hash.